decidim-core/app/controllers/concerns/decidim/amendments_controller.rb

```diff
@@ -42,13 +42,40 @@ def create
       end
     end
 
-    private
-
     def reject; end
 
-    def review; end
+    def review
+      emendation_fields_form = emendation.form.from_model(emendation)
+      params = emendation.attributes
+      params[:id] = emendation.amendment.id
+
+      @form = Decidim::Amendable::ReviewForm.from_params(
+        id: params[:id],
+        amendable_gid: amendable_gid,
+        emendation_gid: params[:emendation_gid],
+        emendation_fields: emendation_fields_form
+      )
+    end
+
+    def accept
+      @form = form(Decidim::Amendable::ReviewForm).from_params(params)
+      enforce_permission_to :accept, :amend, amend: @form.amendable
+      return unless validate(@form)
```

Comment:
Why do we need this?

Reply:
Nice to meet you, @oriolgual . I'm a new member of the coditramuntana decidim team.

The validate command runs the amendable form validations because the new form and the review form doesnt do any of it and they allowed doing crazy things in the create/accept command.
Mostly it does 2 validations:
* amendable_form.invalid?
Enforces the proposal form etiquete.
* emendation_doesnt_change amendable
Prevents the user from creating amendments identical to the proposal.

Its the solution I found at the moment, but maybe you have some suggestions to go about it.